public/sass/elements/_forms.scss

```diff
@@ -3,38 +3,21 @@
 
 // Contents:
 //
-// 1. Helpers
-// 2. Form wrappers
-// 3. Form labels
-// 4. Form hints
-// 5. Form controls
-// 6. Form control widths
+// 1. Form wrappers
+// 2. Form labels
+// 3. Form hints
+// 4. Form controls
+// 5. Form control widths
 
-// 1. Helpers
+// 1. Form wrappers
 // ==========================================================================
 
-// Fieldset is used to group more than one .form-group
-fieldset {
+// .fieldset is used to apply styling to fieldsets and to group more than one .form-group
+.fieldset {
```

Comment:
Are we happy with changes like these? I can see people now writing `<div class="fieldset">` which then breaks accessibility for a bunch of people. Given the overall simplicity of our pages I’m not worried about selector complexity impacting performance so would it be better to have `fieldset.fieldset` as the selector? Anything that enforces accessibility from within our styling is a good idea (imo of course).